docs/specs/biometric.md

```diff
@@ -79,3 +79,15 @@ If the developer really wants the user to sign in different accounts,
 they should prompt the user to disable biometric authentication first.
 
 Disabling biometric authentication deletes the keypair, which requires biometric authentication.
+
+## Configuration
+
+Biometric identity is not shown along with Login ID identity and OAuth identity.
+Biometric identity is shown separately in the settings page.
+By default, biometric identity is hidden.
+
+```yaml
+identity:
+  biometric:
+    is_hidden: true
```

Comment:
We had `list_enabled` in recovery code, do you think this name is suitable here too? Looks more consistent.

Reply:
I was aware of this too. I was not sure whether we would have something else to enable in the future. But you seem agree to use `list_enabled` for consistency. Let me update to use `list_enabled`.